### DISCUSSION TITLE
Generating partly labeled data for ASReview by "An open-source Python framework for systematic review based on PRISMA" (working paper)

### DISCUSSION THREAD

**Discussion by chandraveshchaudhari** [asreview/asreview#839] [Q&A] [answered]
Dear Rens van de Schoot, 

I have been working on writing a paper named "An open-source Python framework for systematic review based on PRISMA". I want to suggest ASReview for active learning screening. The framework will output a file for partly labelled data containing citation columns (such as title and abstract) with an additional column named "label_included".  "label_included" column will contain 0 for irrelevant/excluded, 1 for relevant/included,  empty for no prior knowledge.

Please let me know if ASReview is planning to change the implementation for importing partly labelled data. 

Thanks and regards,
Chandravesh Chaudhari
(Research scholar)
Department of Commerce,
Christ Deemed to be University, Bangalore, India

**Comment by J535D165** [comment, marked as answer]
Dear Chandravesh, Thank you for sharing your work. We are excited to hear about your framework. 

Label_included will be supported in the next releases and therefore compatible with your work.

**Reply by chandraveshchaudhari** [reply]
Thank you for clarifying.
